package.json

```diff
@@ -71,6 +71,7 @@
     "@babel/plugin-transform-runtime": "^7.6.2",
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
+    "@size-limit/preset-big-lib": "^4.0.1",
```

Comment:
If we're using this, doesn't there need to be additional configuration?  And do we really need this?  Seems like a lot of overhead for just checking a bundle size.

Reply:
Size limit will no longer work on it's own - it needs one of it's 3 preset options. This one, `@size-limit/preset-app` or `@size-limit/preset-small-lib` See https://github.com/ai/size-limit#usage

This was the breaking change for `2.0.0` - no need to change our config.